recipes/getmail6/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "getmail6" %}
+{% set version = "6.18.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive//v{{ version }}.tar.gz
+  sha256: 1dda163d20d7c67571d66234d1749e2209bca8cbcb7b944a2381837a82b88661
+
+build:
+  noarch: python
```

Comment:
Looks like it uses scripts and cannot be noarch: https://github.com/getmail6/getmail6/blob/master/setup.py#L84